proto_helper/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+pub struct ProtoBuilder {
+    protoc_search_path: Option<PathBuf>,
+    files: Vec<PathBuf>,
+    import_paths: Vec<PathBuf>,
+    generate_grpc: bool,
+    transformations: Vec<Box<Fn(String) -> String>>,
```

Comment:
I would add a comment explaining what the `transformations` fields are for and also specify the order in which the transformations are applied later. i.e. if `transformations = [T1, T2]` and we later apply them to `code`, would it result in `T1(T2(code))` or `T2(T1(code))`?